zio-cache/shared/src/main/scala/zio/cache/ScopedCache.scala

```diff
@@ -99,230 +99,45 @@ object ScopedCache {
   def makeWith[Key, Environment, Error, Value](
     capacity: Int,
     scopedLookup: ScopedLookup[Key, Environment, Error, Value]
-  )(timeToLive: Exit[Error, Value] => Duration): ZIO[Environment with Scope, Nothing, ScopedCache[Key, Error, Value]] =
+  )(timeToLive: Exit[Error, Value] => Duration): URIO[Environment with Scope, ScopedCache[Key, Error, Value]] =
     makeWith(capacity, scopedLookup, Clock.systemUTC())(timeToLive)
 
   //util for test because it allow to inject a mocked Clock
   private[cache] def makeWith[Key, Environment, Error, Value](
     capacity: Int,
     scopedLookup: ScopedLookup[Key, Environment, Error, Value],
     clock: Clock
```

Comment:
Not related to this PR but can we get rid of this now?

Reply:
Are you speaking about `private[cache] def makeWith[Key, Environment, Error, Value](` ?
If yes no, I need this method for test

Reply:
We should be able to delete it if we just grab the `Clock` from the environment when we construct the cache.

Reply:
https://github.com/zio/zio-cache/pull/133#discussion_r1103009593 I thought about doing that in a separate PR, but if you want I can do it on this PR no problem

Reply:
That's fine!